github-changelog-generator.php

```diff
@@ -1,242 +1,113 @@
 <?php
 
+// "autoload" our classes
+if (!class_exists('ins0\GitHub\Repository')) {
+    require_once 'src/Repository.php';
```

Comment:
That would lead to a BC break as the class then uses classes that wouldn't be loaded without user interaction. Maybe ``v1`` is born then